lib/slacker/application.rb

```diff
@@ -25,12 +26,18 @@ class Application
 set concat_null_yields_null on;
 EOF
 
+    ODBC_Driver = 'odbc'
```

Comment:
Please use Ruby's naming convention for constants (all-caps separated with underscores).
In this case the constant name should be ODBC_DRIVER.